broker.go

```diff
@@ -84,50 +92,44 @@ func handleDefaultPort(w http.ResponseWriter, r *http.Request) {
 }
 
 func handleMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
-	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
+	/* Validation */
 	if m.Author.ID == s.State.User.ID {
-		log.Println("Ignore the bot's message itself")
 		return
 	}
-
-	trimmed := strings.TrimSpace(m.Content)
-
-	// If not in the format "!stonks <ticker>", give up.
-	if !strings.HasPrefix(trimmed, "!stonks ") {
-		log.Println("Ignoring non-prefixed message")
+	if !strings.HasPrefix(strings.TrimSpace(m.Content), "!stonks ") {
 		return
 	}
 
 	ticker := strings.TrimPrefix(m.Content, "!stonks ")
```

Comment:
By doing this, we lose the strings.TrimSpace, which could mean we get really bad trailing whitespace.

Reply:
The original code discard "trimmed" after line 97. Did you mean to carry it forward to line 102?